docs/getting-started/devenv/nodejs-development-tools.md

```diff
@@ -13,7 +13,7 @@ The Node Package eXecute ([`npx`](https://docs.npmjs.com/cli/v10/commands/npx))
 
 It's recommended that you use [Node Version Manager](https://github.com/nvm-sh/nvm) (`nvm`) to install Node.js. This allows you to install and manage specific versions of `node`, which are installed locally in your home directory, avoiding any global permission issues.
 
-Here are the quick instructions for installing `node` using `nvm`. See the [complete installation guide](https://github.com/nvm-sh/nvm#installing-and-updating) for more details.
+Here are the quick instructions for installing `node` using `nvm` and set the recommended Node version for block development. See the [complete installation guide](https://github.com/nvm-sh/nvm#installing-and-updating) for more details.
```

Comment:
Here are the quick instructions for installing `node` using `nvm` and setting the recommended Node.js version for block development.